internal/p2p/network_messenger.go

```diff
@@ -56,13 +80,13 @@ func (m *NetworkMessanger) AnnounceTransactions(txHashes []*chainhash.Hash, peer
 
 // AnnounceTransaction will send an INV message to the provided peers or to selected peers if peers is nil.
 // It will return the peers that the transaction was actually announced to.
-func (m *NetworkMessanger) AnnounceTransaction(txHash *chainhash.Hash, peers []PeerI) []PeerI {
+func (m *NetworkMessenger) AnnounceTransaction(txHash *chainhash.Hash, peers []PeerI) []PeerI {
```

Comment:
We don't use `AnnounceTransaction`, `AnnounceTransactions`, `RequestTransaction` and `RequestTransactions` anywhere. We're only using the one `WithAutoBatch`. 

So maybe we could remove `WithAutoBatch` and make it a default behaviour in these functions above? I find it a bit confusing these methods `WithAutoBatch`. WDYT?